.github/workflows-source/ci-main.yml

```
@@ -199,6 +182,34 @@ jobs:
         run: |
           sudo rm -rf /private/var/tmp/bazel-output
 
+  bazel-test-macos-arm64:
+    name: Bazel Test MacOS Namespace
```

```suggestion
    name: Bazel Test Darwin arm64 [Namespace]
```
For the sake of consistency?